### DISCUSSION TITLE
copilot plugin not enabliing even after successful authentication on intellij for java

### DISCUSSION THREAD

**Discussion by srichys79** [community/community#41849] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

Hi copilot team,

Please help how to enable github copilot for java application on IDE intellij.

i have logged in github and also shows succesful authentication to access github copilot.  but still plugin not enabled.

please help me to configure and enable github-copilot plugin.

**Comment by quzanh1130** [comment]
idk man. i cant help this one :)

**Comment by ghostinhershell** [comment]
Hey there!

Thanks for posting in the GitHub Community, @srichys79! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
